src/app/backend/dashboard.go

```diff
@@ -42,9 +50,18 @@ func main() {
 		log.Print(err)
 	}
 
+	if *argHeapsterHost != "" {
```

Comment:
Can you create heapsterclient.go file and do this logic there? The reason is that we really want to keep main function short and do not contain any logic.